greeclimate/device.py

```diff
@@ -291,7 +307,14 @@ def temperature_units(self, value: int):
     @property
     def current_temperature(self) -> int:
         prop = self.get_property(Props.TEMP_SENSOR)
-        return prop or self.target_temperature
+        if prop:
```

Comment:
consider 0C
please use "if prop is not None:"

Reply:
Done, added a test for this case too.